### DISCUSSION TITLE
Inquiry about Availability of Horizontal Histogram in finplot

### DISCUSSION THREAD

**Discussion by NijanJ** [highfestiva/finplot#448] [General]
I'm curious to know if finplot supports horizontal histograms. Can you confirm if this feature is available or provide any insights into its potential implementation?

Thanks, 
Nijan

**Comment by highfestiva** [comment]
Yes it does. It's clunky due to finplot's time series thinking, unfortunately. Example here: https://github.com/highfestiva/finplot/blob/master/finplot/examples/volume-profile.py
